parlai/crowdsourcing/tasks/turn_annotations_static/json/onboarding_in_flight.jsonl

```diff
@@ -0,0 +1 @@
+{"dialog": [[{"text": "Lorem ipsum dolor sit amet, unum dolor lobortis ad pro", "episode_done": false, "id": "human_evaluator"}, {"text": "Lorem ipsum dolor sit amet, unum dolor lobortis ad pro", "episode_done": false, "id": "model"}], [{"text": "Lorem ipsum dolor sit amet, unum dolor lobortis ad pro", "episode_done": false, "id": "human_evaluator"}, {"text": "Lorem ipsum dolor sit amet, unum dolor lobortis ad pro", "episode_done": false, "id": "model"}], [{"text": "Lorem ipsum dolor sit amet, unum dolor lobortis ad pro", "episode_done": false, "id": "human_evaluator"}, {"text": "Last utterance where user should select bucket_1 and bucket_3", "episode_done": true, "id": "model"}]], "answers": ["bucket_1", "bucket_3"]}
```

Comment:
Hmm, it might be nice to have a note in the README about why the structure of this is different than `onboarding.json`

Reply:
Good catch - changed them both to be Conversations (like). In the case of the regular onboarding, we should change the format to not use the agent_idx perhaps?

Reply:
Yeah, agreed, maybe we can get rid of having to specify agent_idx. Wrote a note to look into this in the future